elasticsearch.md

```diff
@@ -1510,7 +1551,7 @@ readonlyrest:
 
 #### Dynamic variables from JWT claims
 
-The JWT token is an authentication string passed generally as a header or a query parameter to the web browser. If you squint, you can see it's a concatenation of three base64 encoded strings. If you base64 decode the middle string, you can see the "claims object". That is, the object containing the current user's metadata.
```

Comment:
That is + comma means "in other words" i.e.  "Take the smaller number of the list. That is, 0"